### PR TITLE
Check syntax of example configurations

### PR DESCRIPTION
**Pull request by dgl**
Fix a mistake in the hetzner and vultr configs this found.

Signed-off-by: David Leadbeater <dgl@dgl.cx>

### CODE REVIEW

File: documentation/examples/Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: check-examples-syntax
+
+check-examples-syntax: prometheus-*.yml
```
Comment (roidelapluie) [inline documentation/examples/Makefile:16]: Please add .PHONY to both targets

**Review by roidelapluie — COMMENTED** [review]
Awesome

File: documentation/examples/Makefile (hunk 1 above)
Comment (dgl) [reply documentation/examples/Makefile:18]: Done.

**Review by roidelapluie — APPROVED** [review]
LGTM

**Pull request merged**